rules/cloud/azure_kubernetes_secret_or_config_object_access.yml

```diff
@@ -0,0 +1,26 @@
+title: Azure Kubernetes Secret or Config Object Access
+id: 7ee0b4aa-d8d4-4088-b661-20efdf41a04c
+description: Identifies when a Kubernetes account access a sensitve objects such as configmaps or secrets.
+author: Austin Songer
+status: experimental
+date: 2021/08/07
+references:
+    - https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations#microsoftkubernetes
+logsource:
+  service: azure.activitylogs
+detection:
+    selection1:
+          properties.message: MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/CONFIGMAPS/WRITE 
+    selection2:
+          properties.message: MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/CONFIGMAPS/DELETE
+    selection3:
+          properties.message: MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/SECRETS/WRITE
+    selection4:
+          properties.message: MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/SECRETS/DELETE
+    condition: selection1 or selection2 or selection3 or selection4
```

Comment:
more consisted
```yaml
detection:
    selection:
          properties.message: 
            - MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/CONFIGMAPS/WRITE 
            - MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/CONFIGMAPS/DELETE
            - MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/SECRETS/WRITE
            - MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/SECRETS/DELETE
    condition: selection
```
in the log it  `MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/CONFIGMAPS/WRITE` or `Microsoft.Kubernetes/connectedClusters/configmaps/write` ?